docs/source/dummies_network.rst

```diff
@@ -236,52 +236,52 @@ The `transmissions()` function returns a list of transmissions in the network. A
+Note also that the function takes a Node as a parameter. This is the new Node that has been created, and, because Nodes *must* have a Network the node is actually already in the Network. What's happening when this function is called is that the n]Network is being notified that the Node has been added to it and so the Network can take any action that is necessary (e.g. connecting it to other Nodes, sending it Transmissions and so on). Again, we'll see some examples of this shortly and we'll also see how this function is called in specific experiments when we come to the ``Experiment`` class.
 
-The Network class also has a `fail()` and `print_verbose()` function, but these aren't particularly interesting, so let's skip to `calculate_full()`.
+The ``Network`` class also has a ``fail()`` and ``print_verbose()`` function, but these aren't particularly interesting, so let's skip to ``calculate_full()``.
 ::
 
     def calculate_full(self):
         """Set whether the network is full."""
         self.full = len(self.nodes()) >= (self.max_size or 0)
 
-This functions simply tells the network to update the value in its full column to reflect its current size. It is called automaticlaly by Dallinger when new nodes are created so you don't need to worry about it, but its important to know that this function exists and when it is called so you know how Dallinger is keeping track of these things. The goal of this "book" is to pull back the curtain so you get to see Dallinger's inner workings as once you get to that point you'll be able to build new experiments with ease.
+This function simply tells the Network to update the value in its full column to reflect its current size. It is called automatically by Dallinger when new Nodes are created so you don't need to worry about it, but its important to know that this function exists and when it is called so you know how Dallinger is keeping track of these things. The goal of this "book" is to pull back the curtain so you get to see Dallinger's inner workings as once you get to that point you'll be able to build new experiments with ease.
 
 Kinds of Networks
 -----------------
 
-Just like with Nodes, Dallinger comes pre-packaged with a bunch of common networks. You can see them in the file networks.py which is in the same directory as models.py (Dallinger/dallinger/networks.py). Open it up now and find the Chain network:
+Just like with Nodes, Dallinger comes pre-packaged with a bunch of common Networks. You can see them in the file `networks.py` which is in the same directory as `models.py` (`Dallinger/dallinger/networks.py`). Open it up now and find the ``Chain`` network:
 ::
 
 	class Chain(Network):
 
-Note that just like the types of Node it contains a `__mapper_args__` value which is used to fill in the `type` column in the database:
+Note that just like the types of Network it contains a `__mapper_args__` value which is used to fill in the ``type`` column in the database:
```

Comment:
This seems to me like it should be Node, since it seems like its comparing that Node used this __mapper_args__ property?

Reply:
Either or. I put Node initially in reference to the earlier page on Node that ends with a section discussing the various types of Nodes (Agents, and so forth) all of which set the mapper_args value to custom values. But the base Network class also sets it, so that works too.

Reply:
I think what threw me here, is that `types of Network` is not mentioned on that page. So `just like' had me thinking, what is he comparing this to etc